astropy/wcs/tests/test_wcs.py

```diff
@@ -1183,3 +1183,22 @@ def test_footprint_contains():
 
     hasCoord = test_wcs.footprint_contains(SkyCoord(24,2,unit='deg'))
     assert hasCoord == False
+
+
+def test_cunit():
+    # Initializing WCS
+    w1 = wcs.WCS(naxis=2)
+    w2 = wcs.WCS(naxis=2)
+    w3 = wcs.WCS(naxis=2)
+    # Initializing the values of cunit and ctype
+    w1.wcs.cunit = ['deg', 'm/s']
+    w2.wcs.cunit = ['km/h', 'km/h']
+    w3.wcs.cunit = ['deg', 'm/s']
+    w2.wcs.ctype = ["RA---CAR", "DEC--CAR"]
+
+    assert w1.wcs.cunit == w1.wcs.cunit
+    assert w1.wcs.cunit == w3.wcs.cunit
+    assert w1.wcs.cunit != w2.wcs.cunit
+    assert w1.wcs.cunit != w2.wcs.ctype
```

Comment:
This is hard to miss. Could you compare it to a literal list and add a Comment?

Reply:
You could drop this test. It should be identical to the following test.

Reply:
I did not understand which one you want to drop `assert w1.wcs.cunit != w2.wcs.ctype` here I compared two different classes

Reply:
Yes, the test comparing `cunit` to `ctype` - you also have the comparison to a `list`. That should be enough and is much easier to understand.